runtimes/typescript/lbr-plutus/build.nix

```diff
@@ -0,0 +1,15 @@
+{ ... }:
+{
+  perSystem = { config, ... }:
+    let
+      typescriptFlake =
+        config.overlayAttrs.extras.typescriptFlake {
+          name = "lbr-plutus-typescript";
```

Comment:
I wonder if we should rather name this `lbr-plutus` and let typescriptFlake add the `-typescript` suffix. I think that's what we do elsewhere?